persistent-sqlite/persistent-sqlite.cabal

```diff
@@ -35,6 +35,12 @@ flag have-usleep
 flag json1
   description: Enable json1 in the vendored SQLite library; has no effect if a system SQLite library is used.
   default: True
+flag use-stat3
+  description: Enable STAT3 in the vendored SQLite library; has no effect if a syste
```

Comment:
looks like this was cut off?

Reply:
Yeah, I was manually cherrypicking and made a copy/paste fail, fixed in the new commit.